### PR TITLE
Adding missing, updating outdated functions

### PR DESCRIPTION
**Pull request by Neodevils**
I'll be updating function files day-by-day.

### CODE REVIEW

**Comment by Neodevils** [comment]
Deleted $complexCooldown on the summary, and added $advanceCooldown.

**Review by Leref — APPROVED** [review]

**Comment by Neodevils** [comment]
$if function slightly got better details for bot developers understand it.

**Comment by Neodevils** [comment]
Cause of using an app to "push", it sends two commits sometimes. Don't mind it.

File: src/functions/usdcreateapplicationcommand.md [hunk 1]
```diff
@@ -12,26 +12,31 @@ Create an application command which is can be:
 ### Usage
 
 ```php
-$createApplicationCommand[guildID/global;application name;application description?;default permission;type;options?]
```
Comment (Leref) [inline src/functions/usdcreateapplicationcommand.md]: Revert this.
It should be guildID or global.

File: src/functions/usddeleteapplicationcommand.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+Delete an application command which is can be:
+* Slash type
+* Message type
+* User type
+
+### Usage
+
+```php
+$deleteApplicationCommand[location;application id]
```
Comment (Leref) [inline src/functions/usddeleteapplicationcommand.md]: Should be guildID or global in location.

File: src/functions/usdgetapplicationcommandid.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+Get an application command which is can be:
+* Slash type
+* Message type
+* User type
+
+### Usage
+
+```php
+$getApplicationCommand[application name;location]
```
Comment (Leref) [inline src/functions/usdgetapplicationcommandid.md]: Incorrect usage name, it's missing `id` -> `$getApplicationCommandID`

**Review by Leref — CHANGES_REQUESTED** [review]
Please fix this accordingly.

**Review by Leref — APPROVED** [review]

**Pull request merged**